Datahub.Core/Services/SystemNotificationService.cs

```diff
@@ -143,29 +143,20 @@ private async Task DoNotifyUsers(IEnumerable<string> userIds)
             }
         }
 
-        private static Expression<Func<SystemNotification, bool>> CreateUserNotificationCondition(string userId, bool unreadOnly)
```

Comment:
Is there any reason to remove this function? Having it here ensures that it's always the same condition for the count and results, and if it ever needs to be updated it won't get out of sync.

Reply:
Ping Yannick on this one - I am not sure about the rationale. It's a private one, not attached to either way. We can rollback that one if necessary.